TuneUpTests/TuneUpTests.csproj

```diff
@@ -53,39 +55,50 @@
     <Reference Include="System.Xaml" />
     <Reference Include="WindowsBase" />
   </ItemGroup>
+
+  <!-- Source code files to be compiled -->
   <ItemGroup>
     <Compile Include="TuneUpTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
+
+  <!-- Reference to the main project -->
   <ItemGroup>
     <ProjectReference Include="..\TuneUp\TuneUp.csproj">
       <Project>{6ff12d3a-025e-49a5-a773-d99ab82778a3}</Project>
       <Name>TuneUp</Name>
     </ProjectReference>
   </ItemGroup>
+
+  <!-- Package references -->
   <ItemGroup>
-    <PackageReference Include="DynamoVisualProgramming.Core">
-      <Version>2.12.0.5650</Version>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.DynamoServices">
-      <Version>2.12.0.5650</Version>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.Tests">
-      <Version>2.12.0.5650</Version>
-    </PackageReference>
-    <PackageReference Include="DynamoVisualProgramming.ZeroTouchLibrary">
-      <Version>2.12.0.5650</Version>
-    </PackageReference>
+    <PackageReference Include="DynamoVisualProgramming.Core" Version="3.2.1.5366">
+      <ExcludeAssets>runtime</ExcludeAssets>
+	  <IncludeAssets>compile; build; native; contentfiles; analyzers</IncludeAssets>
```

Comment:
same comment as above here as well.

Reply:
@reddyashish , hope the latest commit is an improvement